app.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+const app = express()
+
+
+app.listen(3000)
```

Comment:
You need to include this alternative in case port has been specified like `process.env.PORT || 3000`

Reply:
This is done!